addons/fools/index.js

```diff
@@ -0,0 +1,265 @@
+export default async function ({ addon, global, console }) {
+  document.addEventListener("keydown", async (e) => {
+    if (e.key == "s") {
+      await setup();
+    }
+  });
+}
+const wait = (s) =>
+  new Promise((resolve, reject) => {
+    setTimeout(() => {
+      return resolve();
+    }, s);
+  });
+async function setup() {
+  document.querySelector("#view > div > div:nth-child(3) > div:nth-child(1)").scrollIntoView();
```

Comment:
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'scrollIntoView')

Reply:
lmao well that's interesting

Reply:
works on my device, lemme figure out why

Reply:
for now test while logged out, i have some dumb selectors

Reply:
> Uncaught (in promise) TypeError: Cannot read properties of null (reading 'scrollIntoView')

got the same